client/src/app/site/guards/permission.guard.ts

```diff
@@ -13,26 +14,25 @@ export class PermissionGuard implements CanLoad {
         private router: Router,
         private reroute: RerouteService,
         private osRouter: OpenSlidesRouterService,
+        private operator: OperatorService,
         private authCheck: AuthCheckService
     ) {}
 
-    public async canLoad(route: Route, segments: UrlSegment[]): Promise<boolean> {
+    public async canLoad(route: Route, segments: UrlSegment[]): Promise<boolean | UrlTree> {
         const url = this.getCurrentNavigationUrl();
         if (this.osRouter.isOrganizationUrl(url)) {
             if (!(await this.authCheck.isAuthorizedToSeeOrganization())) {
-                this.reroute.forwardToOnlyMeeting(url === `/info` ? [`info`] : []);
-                return false;
+                const meetingId = this.operator.onlyMeeting.toString();
```

Comment:
Why not continue using forwardToOnlyMeeting here?
Haven't you changed it to do exactly what you are also doing here?